tests/periph_rtc/Makefile

```diff
@@ -0,0 +1,8 @@
+export APPLICATION = periph_rtc
+include ../Makefile.tests_common
+
+FEATURES_REQUIRED = periph_rtc
```

Comment:
You also need and entry `FEATURES_PROVIDED += periph_rtc` in boards/stm32f0discovery/Makefile.features